build.gradle

```diff
@@ -21,9 +21,9 @@ plugins {
 
 }
 
+
 import com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar
 import de.undercouch.gradle.tasks.download.Download
-import org.gradle.internal.os.OperatingSystem
```

Comment:
oh, huh, for some reason intellij fully qualified the names below and dropped the import...  Will fix